endpoints/s3/src/completemultipartupload.cpp

```diff
@@ -92,44 +107,42 @@ void irods::s3::actions::handle_completemultipartupload(
 		}
 	}
 
-	logging::debug("{} s3_bucket={} s3_key={}", __FUNCTION__, s3_bucket.string(), s3_key.string());
+	logging::debug("{} s3_bucket={} s3_key={}", __func__, s3_bucket.string(), s3_key.string());
 
 	fs::path path;
 	if (auto bucket = irods::s3::resolve_bucket(url.segments()); bucket.has_value()) {
 		path = bucket.value();
 		path = irods::s3::finish_path(path, url.segments());
-		logging::debug("{}: CompleteMultipartUpload path={}", __FUNCTION__, path.string());
+		logging::debug("{}: CompleteMultipartUpload path={}", __func__, path.string());
 	}
 	else {
-		logging::error("{}: Failed to resolve bucket", __FUNCTION__);
+		logging::error("{}: Failed to resolve bucket", __func__);
 		response.result(beast::http::status::forbidden);
-		logging::debug("{}: returned {}", __FUNCTION__, response.reason());
+		logging::debug("{}: returned {}", __func__, response.reason());
 		session_ptr->send(std::move(response));
 		return;
 	}
 
-	fs::client::create_collections(conn, path.parent_path());
```

Comment:
This is a consequence of the multipart enhancement.  We now write parts directly to iRODS during part uploads, if possible.  Worst case scenario is where part 1 was uploaded last.  In this case it would be the only part written directly to iRODS.  So by the time we get here the parent collection would have already been created.

Reply:
Okay, makes sense. Resolving this.